cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -259,6 +259,22 @@
   </property>
 
   <property>
+    <name>ssl.internal.enabled</name>
```

Comment:
After discussions with @sreevatsanraman and @nitinmotgi we decided to go with two flags.